src/drivers/pca9685_pwm_out/module.yaml

```diff
@@ -4,24 +4,75 @@ actuator_output:
     - param_prefix: PCA9685
       channel_label: 'Channel'
       standard_params:
-        disarmed: { min: 800, max: 2200, default: 1000 }
-        min: { min: 800, max: 1400, default: 1000 }
-        max: { min: 1600, max: 2200, default: 2000 }
-        failsafe: { min: 800, max: 2200 }
+        disarmed: { min: 0, max: 4096, default: 1000 }
+        min: { min: 0, max: 4096, default: 1100 }
+        max: { min: 0, max: 4096, default: 1900 }
+        failsafe: { min: 0, max: 4096 }
```

Comment:
This used to be the range 800...2200 to accommodate ESCs that are controlled by a PWM signal. Why change this to 0...4096?

Reply:
Yes it is capable to output in a totally different manner, according to:

> PCA9685 driver has one extra flag that will put the channel into Duty-Cycle output mode, if enabled.
> 
> When Duty-Cycle mode is enabled for one channel:
> 
> - This channel of PCA9685 becomes an "analog output", which can be used to blink an LED, or drive a L298N based brushed motor, etc.
> - min, max and disarmed value can be set between 0 and 4096, corresponding to no output (always low) and full output (always high)
> - if all channels are put into Duty-Cycle mode, then the maximum PWM frequency of 1536 can be set
> 
> Otherwise its output can be used to drive an ESC or servo, and PWM frequency is limited up to 400Hz.

Reply:
I see. Still, could we keep the default values of 1000 / 2000?

Reply:
> I see. Still, could we keep the default values of 1000 / 2000?

I have no strong opinion against that... @dagar any suggests?